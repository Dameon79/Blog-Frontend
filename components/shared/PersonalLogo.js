import { Component } from 'react'
import Link from 'next/link'


class PersonalLogo extends Component {
  render () {
    return (
    <Link href="/index">
      <img src='/static/images/logo.png' className="c-personal-logo"></img>
    </Link>
    );
  }
}

export default PersonalLogo