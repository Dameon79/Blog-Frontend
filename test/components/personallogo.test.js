import PersonalLogo from '../../components/shared/PersonalLogo'

describe('<PersonalLogo />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <PersonalLogo />
    )

    expect(component).toMatchSnapshot()
  })
})