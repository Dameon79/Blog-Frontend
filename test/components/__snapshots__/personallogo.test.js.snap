// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PersonalLogo /> it renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PersonalLogo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <img
        className="c-personal-logo"
        src="/static/images/logo.png"
      />,
      "href": "/index",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "c-personal-logo",
        "src": "/static/images/logo.png",
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <img
          className="c-personal-logo"
          src="/static/images/logo.png"
        />,
        "href": "/index",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "c-personal-logo",
          "src": "/static/images/logo.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
